MANIFEST.in

```diff
@@ -1 +1,7 @@
-include README.rst LICENSE *.txt
\ No newline at end of file
+include README.rst LICENSE requirements*.txt
```

Comment:
It's probably missing `test_requirements.txt`, isn't it?

Reply:
Ya, whoops. Fixed.